doc/os_requirements.md

```diff
@@ -40,3 +40,11 @@ Edgehog Device Runtime has a number of requirements in order to provide device m
   ```sh
   cargo build --features systemd
   ```
+
+### Remote Terminal
+
+- **[ttyd](https://github.com/tsl0922/ttyd)**: command-line tool for sharing terminal over the web. 
+  To provide the remote terminal functionality, the program must run on the default port `7681` with
+  the flag `-W` enabled, which enables the write-mode on the remote terminal. 
+  To start ttyd locally run the command `ttyd -W bash`.
+  Note: It is possible to specify a different kind of shell, depending on the ones installed.
```

Comment:
Move into optional features and add how to build with `forwarder` feature enabled.
```suggestion
- **[ttyd](https://github.com/tsl0922/ttyd)**: command-line tool for sharing terminal over the web. 
  To provide the `Remote Terminal` functionality, the program must run on the default port `7681` with
  the flag `-W` enabled, which enables the write-mode on the remote terminal. 
  To start ttyd locally run the command `ttyd -W bash`.
  Note: It is possible to specify a different kind of shell, depending on the ones installed.
  ```sh
  cargo build --features forwarder
  ```
```